src/components/atoms/UiTextarea/UiTextarea.vue

```diff
@@ -136,9 +136,9 @@ const setTextareaSize = (mutationList: MutationRecord[]) => {
   const calcMinHeightPx = Number(minHeight.replace('px', '')) + (2 * Number(textareaMargin.replace('px', '')));
   const calcHeight = calcHeightPx > calcMinHeightPx ? null : `${calcMinHeightPx}px`;
 
-  textareaSize.width = width;
-  textareaSize.height = calcHeight;
-  textareaSize.minHeight = minHeight;
+  textareaSize.width = width || undefined;
+  textareaSize.height = calcHeight || undefined;
+  textareaSize.minHeight = minHeight || undefined;
```

Comment:
Is this necessary? What is the value of `width`, `calcHeight`, and `minHeight` if it is not in the `style`? Isn't it `undefined` indeed?

Reply:
TBH they return an empty string 🙈 :thank_you: